contrib/babelfishpg_tsql/sql/babelfishpg_tsql.in

```diff
@@ -15,8 +15,8 @@ SELECT set_config('search_path', 'sys, '||current_setting('search_path'), false)
 #include "sys_cast.sql"
 #include "coerce.sql"
 #include "ownership.sql"
-#include "information_schema_tsql.sql"
 #include "sys_views.sql"
+#include "information_schema_tsql.sql"
```

Comment:
Earlier when we were using all_objects for joins in routines view it was giving error because we were executing information_schema_tsql.sql first and then execute sys_views.sql(where sys.all_objects is defined).
So I changed the order but later we removed that join and hence changed this order to as it was previously.